### PR TITLE
Log files tutorial

### PR DESCRIPTION
**Pull request by IvanFon**

### CODE REVIEW

File: tutorials/desktop/decoding-log-files/decoding-log-files.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+Contains login info used by other utilities to find out who's logged in. To view currently logged in users, use the `who` command.
+
+positive
+: **This is not a complete list!**
+You can search the web for more locations relevant to what you're trying to debug. There is also a slightly more complete list [here](https://help.ubuntu.com/community/LinuxLogFiles#System_Logs).
+
+## Viewing logs using GNOME System Log Viewer
+Duration: 3:00
+
+The GNOME System Log Viewer provides a simple GUI for viewing and monitoring log files. It comes pre-installed with Ubuntu under the name `System Log`.
```
Comment (caldav) [inline tutorials/desktop/decoding-log-files/decoding-log-files.md]: Maybe add that on Ubuntu 17.10 and above, it's called "Logs"

**Bold**, not `code` for app names.

File: tutorials/desktop/decoding-log-files/decoding-log-files.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+### System Log Viewer Interface
```
Comment (caldav) [inline tutorials/desktop/decoding-log-files/decoding-log-files.md]: interface

File: tutorials/desktop/decoding-log-files/decoding-log-files.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+![GNOME System Log Viewer Interface](images/log-viewer-1.png)
+
+The log viewer has a simple interface. The sidebar on the left shows a list  of open log files, with the actual contents of the currently shown file displayed on the right.
```
Comment (caldav) [inline tutorials/desktop/decoding-log-files/decoding-log-files.md]: There is an extraneous space between "a list" and "of"

Keep it simple, what else would it be? :smile:  : "actual contents" -> "contents" 

currently shown -> selected

File: tutorials/desktop/decoding-log-files/decoding-log-files.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+Clicking on the cog at the top right of the window will open a menu allowing you to change some display settings, as well as open and close log files.
+
+There is also a magnifying glass icon to the right of the cog that allows you to search within the currently selected log file.
+
+### More information
+
+If you wish to learn more about the GNOME System Log Viewer, you may visit the [official documentation](https://help.gnome.org/users/gnome-system-log/).
+
+## Viewing and monitoring logs in the command line
```
Comment (caldav) [inline tutorials/desktop/decoding-log-files/decoding-log-files.md]: in -> from

File: tutorials/desktop/decoding-log-files/decoding-log-files.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+Duration: 5:00
+
+It is also important to know how to view logs in the command line. This is especially useful when you're remotely connected to a server and don't have a GUI.
+
+There are some commands that will be useful to us when working with log files in the command line.
```
Comment (caldav) [inline tutorials/desktop/decoding-log-files/decoding-log-files.md]: Maybe a simpler sentence: "The following commands will be useful when working with log files."

File: tutorials/desktop/decoding-log-files/decoding-log-files.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+
+### Viewing files
+
+The most basic way to view files from the command line is using the `cat` command. You simply pass in the filename, and it outputs the entire contents of the file to the command line: `cat file.txt`.
```
Comment (caldav) [inline tutorials/desktop/decoding-log-files/decoding-log-files.md]: the entire contents of the file ~~to the command line~~

File: tutorials/desktop/decoding-log-files/decoding-log-files.md [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+
+We may also want to quickly view the first or last `n` lines of a file. This is where the `head` and `tail` commands come in handy. These commands work much like `cat`, although you can specify how many lines from the start/end of the file you want to view. To view the first 15 lines of a file, we run `head -n 15 file.txt`, and to view the last 15, we run `tail -n 15 file.txt`. Due to the nature of log files being appended to at the bottom, the `tail` command will generally be more useful.
+
+### Monitoring files
+
+To monitor a log file, you may pass the `-f` flag to `tail`. It will keep running, printing new additions to the file, until you stop it (Ctrl + C). For example: `tail -f file.txt`.
+
+### Searching files
+
+One way that we looked at to search files is to open the file in `less` and press `/`. A faster way to do this is to use the `grep` command. We specify what we want to search for in double quotes, along with the filename, and `grep` will print all the lines containing that search term in the file. For example to search for lines containing "test" in `file.txt` `grep "test" file.txt`.
```
Comment (caldav) [inline tutorials/desktop/decoding-log-files/decoding-log-files.md]: The two inline code blocks next to each other look a bit off when rendered by the tutorial engine, you can fix it with: 

"in `file.txt`, you can run `grep "test" file.txt`."

File: tutorials/desktop/decoding-log-files/decoding-log-files.md [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+If the result of a `grep` search is too long, you may pipe it to `less`, allowing you to scroll and search through it: `grep "test" file.txt | less`.
+
+### Editing files
+
+The simplest way to edit files from the command line is to use `nano`. It is a very simple editor, which has all the most useful keybinds directly on screen. To run it, just give it a filename (`nano file.txt`). To close or save a file, press Ctrl + X. The editor will ask you if you want to save your changes. Press `y` for yes or `n` for no. If you choose yes, it will ask you for the filename to save the file as. If you are editing an existing file, the filename will already be there. Simply leave it as it is and it will save to the proper file.
+
+## Conclusion
+Duration: 1:00
+
+Congratulations, you now have enough knowledge of log file locations, usage of the GNOME System Log Viewer and basic command line commands to properly monitor and trouble-shoot problems that arise on your system.
```
Comment (caldav) [inline tutorials/desktop/decoding-log-files/decoding-log-files.md]: I won't block on that, but a longer conclusion with links to resources to learn more would be nice.

**Comment by IvanFon** [comment]
Thank you for the in-depth review!

I've made the requested changes. Please let me know if there's anything I missed.

**Pull request merged**